build-system/tasks/visual-diff/index.js

```diff
@@ -57,11 +57,18 @@ const percyCss = [
   '.i-amphtml-new-loader * { animation: none !important; }',
 ].join('\n');
 
-// Pin the version of Chromium to 78.0.3904.0, same as the one running on Percy.
-// Use https://omahaproxy.appspot.com/ to convert version<->revision numbers.
-// REPEATING TODO(@ampproject/wg-infra): keep this pinned with Percy whenever we
-// update the version of Chrome in the project settings.
-const PUPPETEER_CHROMIUM_REVISION = '693954';
+// REPEATING TODO(@ampproject/wg-infra): Update this whenever the Percy backend
+// starts using a new version of Chrome to render DOM snapshots.
+//
+// Steps:
+// 1. Open a recent Percy build, and click the “ⓘ” icon
+// 2. Note the Chrome major version at the bottom
+// 3. Look up the full version at https://en.wikipedia.org/wiki/Google_Chrome_version_history
+// 4. Open https://omahaproxy.appspot.com in a browser
+// 5. Go to "Tools" -> "Version information"
+// 6. Paste the full version in the "Version" field and click "Lookup"
+// 7. Copy the value next to "Branch Base Position" and update the line below
+const PUPPETEER_CHROMIUM_REVISION = '812852'; // 87.0.4280.0
```

Comment:
Nit: Could we work with Percy to get an API for this?

Reply:
I agree, we shouldn't have to do this. @danielrozenberg, is this override even needed now that Percy is using v87?

Also, can we ask Percy about updating their version of Chrome more often because our tests frequently depend on new browser functionality?

Reply:
> Nit: Could we work with Percy to get an API for this?

That would be perfect, I'll check on this

> is this override even needed now that Percy is using v87?

I say yes - Chrome will probably continue updating and adding new features faster than Percy will change their backing rendering engine, and we always want to be 

> Also, can we ask Percy about updating their version of Chrome more often because our tests frequently depend on new browser functionality?

I already asked, their reply was "we're working on this" :)